packages/playwright-test/src/plugins/webServerPlugin.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import type { TestPlugin, WebServerConfig } from '../types';
+import { WebServer } from '../webServer';
+
+export const webServer = (config: WebServerConfig): TestPlugin => {
+  // eslint-disable-next-line no-console
+  const server = new WebServer(config, { onStdOut: console.log, onStdErr: console.error });
```

Comment:
While this is not ideal, this is the same behavior for logs in globalSetup/globalTeardown.

We don't want to do OOP, so there's not a clear story for package up these logs. (Other than possible passing a reporter in…but since plugins can also configure reporters, that can get messy real quick. I'm willing to say: this output isn't critical, and if you need it, it's in your CI/terminal logs.)

Reply:
This is fine for this PR, but I think we can do something like `TestPlugin.setLoggers()` or similar and resolve this issue. WDYT?

Reply:
Sure— @pavelfeldman would this help with the vite debug story as well? Does vite allow you to set a logger, or will it just spew to stderr/stdout?